recipe/meta.yaml

```diff
@@ -11,30 +15,36 @@ source:
     sha256: 69e81178839594049d84526b8f9612825f238f7cfe1f6d579bd6461ed48e5aa0
 
 build:
-  number: 0
+  number: {{ build_number }}
   skip: true  # [win or osx]
+  features:
+    - blas_{{ variant }}
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
+    - pkgconfig
   host:
     - cython >=0.24
-    - pip
+    - setuptools
```

Comment:
Any reason for dropping `pip`?

Reply:
I dropped because ``pip`` is not being used in ``mlpack`` and according to the documentation is just necessary ``setuptools``